plugins/kubernetes-backend/src/kubernetes-auth-translator/TokenKubernetesAuthTranslator.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { KubernetesAuthTranslator } from './types';
+import { ClusterDetails } from '../types/types';
+import { KubernetesRequestAuth } from '@backstage/plugin-kubernetes-common';
+
+export class TokenKubernetesAuthTranslator implements KubernetesAuthTranslator {
```

Comment:
Agreed that this format is better - however, I copied this file from the [`OidcKubernetesAuthTranslator`](https://github.com/backstage/backstage/blob/master/plugins/kubernetes-backend/src/kubernetes-auth-translator/OidcKubernetesAuthTranslator.ts), and decided at the time that it would be better to keep all auth translators as similar as possible, for consistency. 

Not sure if it would make sense to change this auth translator but keep the others the same, or change the other auth translators to match this new format as part of this specific PR.